### PR TITLE
C3-803: Rebrand the Certified Hardware Details Page

### PR DESCRIPTION
**Pull request by andrejvelichkovski**
## Done

This PR rebrands the `ubuntu.com/certified/<CANONICAL_ID>/<RELEASE>` view to follow the new Vanilla framework guidelines.

Few examples of the website:
* https://ubuntu.com/certified/202001-27687/18.04%20LTS
* https://ubuntu.com/certified/202102-28706/22.04%20LTS
* https://ubuntu.com/certified/202106-29147/20.04%20LTS

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Demo link: https://ubuntu-com-14466.demos.haus/

## Issue / Card

Fixes # C3-803

### Design in Figma
image

### Implemented view

image

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/certified/hardware-details.html
Comment (andrejvelichkovski) [inline templates/certified/hardware-details.html:1]: This used to be the old template rendered. I removed it and replaced it with a separate directory `templates/certified/hardware-details` where I put all the files required for this view.

File: templates/certified/hardware-details/_hero-section.html
Comment (andrejvelichkovski) [inline templates/certified/hardware-details/_hero-section.html:1]: This file renders the hero section of the web page
image

File: templates/certified/hardware-details/_hardware-table.html
Comment (andrejvelichkovski) [inline templates/certified/hardware-details/_hardware-table.html:1]: This file renders the hardware details table
image

**Comment by akbarkz** [comment]
Demo instance can be accessed by this link: https://ubuntu-com-14466.demos.haus/

**Comment by juanruitina** [comment]
LGTM, UX+1!

**Review by anthonydillon — APPROVED** [review]
Code and QA LGTM 👍

**Pull request merged**